backend/controllers/whereabouts/scheduledMoves.ts

```diff
@@ -24,6 +21,98 @@ export default ({ prisonApi }) => {
           text: values.description,
         }))
         .sort(sortTextAlphabetically),
+      courtEvents,
+    })
+
+  const index = async (req, res) => {
+    const { userDetails } = req.session
+    const { activeCaseLoadId } = userDetails
+    const date = req.query?.date || moment().format('DD/MM/YYYY')
+
+    const movementReasons = await prisonApi.getMovementReasons(res.locals)
+    const agencyDetails = await prisonApi.getAgencyDetails(res.locals, activeCaseLoadId)
+
+    const transfers = await prisonApi.getTransfers(res.locals, {
+      fromDateTime: isoDateTimeStartOfDay(date, 'DD/MM/YYYY'),
+      toDateTime: isoDateTimeEndOfDay(date, 'DD/MM/YYYY'),
+      agencyId: activeCaseLoadId,
+      courtEvents: true,
+      transferEvents: true,
+      releaseEvents: true,
+    })
+
+    const offenderNumbers = [
```

Comment:
won't you end up with duplicates here?  Would be good to therefore remove by putting into a set - maybe something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#relation_with_array_objects?